exec/cnex/global.c

```diff
@@ -89,15 +90,15 @@ void global_callFunction(const char *pszFunc, TExecutor *exec)
     fatal_error("global_callFunction(): \"%s\" - invalid or unsupported predefined function call.", pszFunc);
 }
 
-void print(TExecutor *exec)
+void print(struct tagTExecutor *exec)
```

Comment:
I don't understand why you keep changing these. What's the point of having the typedef to TExecutor if you're not going to use it? If you're going to use "struct tagTExecutor" everywhere, I'd rather see the typedef removed completely.

Reply:
They are used by type everywhere in the code, except function calls.  This keeps my includes clean, so I don't have to include everything in every .h file.  I'd like to keep the typedefs, but at the same time, I can see removing them.  It just takes a lot more code to define them when I need them.